portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,62 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Enrique's Portfolio</title>
+     <!-- Bootstrap CSS CDN -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.0/css/bootstrap.min.css" integrity="sha384-9gVQ4dYFwwWSjIDZnLEWnxCjeSWFphJiwGPXr1jddIhOegiu1FwO5qRGvFXOdJZ4" crossorigin="anonymous">
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <header class="header container-fluid">
+        <div class="overlay"></div>
+        <div class="welcome">
+            <h1>Hello, my name is Enrique Aviña</h1>
+                <a href="#about">Click here to see my work</a>
+        </div>
+  </header>
+
+  <div id="scrollable">
+    <div id="navbar">
+      <div class="flex-parent" id="sticky-container">
+        <a href="#about">About</a>
+        <a href="#projects">Projects</a>
+        <a href="#interests">Interests</a>
+      </div>
     </div>
+    <div class="section" id="about">
+        <h1>About me!</h1>
+        <img src="images/me.jpg"/>
+        <p>Hello! My name is Enrique Aviña, I am a rising junior studying computer science at MIT and I'm very interested in robotics, simulation, and public policy. I'm originally from Redwood City, California and have a twin brother (although we don't look all that alike). I love soccer, and have played on a team since I was 7, and love FIFA too! 
+            Scroll down to see some of my projects and hobbies!</p>
+    </div>
+    <div class="section" id="projects">
+         <h1>Projects</h1>
+         <p>Please enjoy these projects from my (very) outdated <a href="http://rickyavina.com/index.html#projects">personal website</a></p>
+         <p>I am currently very interested in machine learning, and reinforcement learning in particular. I'd love to highlight one project.
```

Comment:
As discussed during 1:1, it is better to use bullet point to high-light the keywords instead of a large paragraph.